include/k4a/k4a.h

```diff
@@ -1985,6 +1985,88 @@ K4A_EXPORT k4a_result_t k4a_transformation_depth_image_to_color_camera(k4a_trans
                                                                        const k4a_image_t depth_image,
                                                                        k4a_image_t transformed_depth_image);
 
+/** Transforms depth map and a custom image into the geometry of the color camera.
+ *
+ * \param transformation_handle
+ * Transformation handle.
+ *
+ * \param depth_image
+ * Handle to input depth image.
+ *
+ * \param custom_image
+ * Handle to input custom image.
```

Comment:
In the resolution of the depth image, right? If we expect certain key properties set for this image, we should be clear about it here.

Reply:
I can add a remark about that.